test/common/config/udpa_resource_test.cc

```diff
@@ -167,10 +165,8 @@ TEST(UdpaResourceLocatorTest, Schemes) {
     EXPECT_EQ("foo", resource_locator.authority());
     EXPECT_EQ("bar", resource_locator.resource_type());
     EXPECT_EQ(2, resource_locator.id().size());
```

Comment:
No need for the assertion on size, since that's already implicitly covered by the `ElementsAre()` check below.